Sources/SwiftFormatWhitespaceLinter/WhitespaceLinter.swift

```diff
@@ -414,21 +414,21 @@ public class WhitespaceLinter {
 
 extension Diagnostic.Message {
   static let trailingWhitespaceError = Diagnostic.Message(
-    .warning, "[TrailingWhitespace]: remove trailing whitespace.")
+    .warning, "[TrailingWhitespace]: remove trailing whitespace")
 
   static func indentationError(_ spaces: Int) -> Diagnostic.Message {
-    return .init(.warning, "[Indentation]: indentation should be \(spaces) spaces.")
+    return .init(.warning, "[Indentation]: indentation should be \(spaces) spaces")
   }
 
   static func spacingError(_ spaces: Int) -> Diagnostic.Message {
-    return .init(.warning, "[Spacing]: should be \(spaces) spaces.")
+    return .init(.warning, "[Spacing]: should be \(spaces) spaces")
   }
 
-  static let removeLineError = Diagnostic.Message(.warning, "[RemoveLine]: remove line break.")
+  static let removeLineError = Diagnostic.Message(.warning, "[RemoveLine]: remove line break")
 
   static func addLinesError(_ lines: Int) -> Diagnostic.Message {
-    return .init(.warning, "[AddLines]: add \(lines) line breaks.")
+    return .init(.warning, "[AddLines]: add \(lines) line breaks")
   }
 
-  static let lineLengthError = Diagnostic.Message(.warning, "[LineLength]: line is too long.")
+  static let lineLengthError = Diagnostic.Message(.warning, "[LineLength]: line is too long")
```

Comment:
This is a weird one too—I'm not sure how we make it imperative unless we say "shorten the line", which seems demonstrably worse. It's probably best left as is.

Reply:
I spent some time thinking about how to make this imperative, but I couldn't come up with anything that I felt was an improvement. It's difficult to state what exactly needs to change.

I think the current text is a good compromise compared to trying to make a very vague imperative statement (e.g. "make this line shorter") or trying to figure out a specific change to make (e.g. "add a line break at column N"). I vote for keeping the current wording.